resources/content/schema/base/webhook.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
So it doesn't work unless you add this? I expected it to work without this file.